runtime/src/lib.rs

```diff
@@ -186,6 +189,23 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
 }
 
 pub fn handle_mailbox_commands(drivers: &mut Drivers) -> CaliptraResult<()> {
+    #[cfg(feature = "riscv")]
+    // TODO implement in emulator
+    fn setup_mailbox_wfi(drivers: &mut Drivers) {
+        use caliptra_drivers::IntSource;
+
+        caliptra_cpu::csr::mie_enable_external_interrupts();
+
+        // Misc clock gating breaks the state of MEPC so disable it
+        caliptra_cpu::csr::mcgc_disable_misc_cg();
+
+        // Set highest priority so that Int can wake CPU
+        drivers.pic.int_set_max_priority(IntSource::SocIfcNotif);
+        drivers.pic.int_enable(IntSource::SocIfcNotif);
+
+        drivers.soc_ifc.enable_mbox_notif_interrupts();
+    }
```

Comment:
nit: I would prefer making this a top level helper rather than defining a function inside this funciton

Reply:
Done